packages/sui-studio-create/src/index.js

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env node
+/* eslint no-console:0 */
+
+const colors = require('colors')
+const program = require('commander')
+const BASE_DIR = process.cwd()
+const fse = require('fs-extra')
+
+program
+  .on('--help', () => {
+    console.log('  Examples:')
+    console.log('')
+    console.log('    $ sui-studio-create <project-name>')
+    console.log('    $ sui-studio-create sui-studio')
+    console.log('    $ sui-studio-create --help')
+    console.log('    $ sui-studio-create -h')
+    console.log('')
+  })
+  .parse(process.argv)
+
+const [PROJECT_NAME] = program.args
+
+const showError = (msg) => {
+  program.outputHelp(txt => colors.red(txt))
+  console.error(colors.red(msg))
+  process.exit(1)
+}
+
+const writeFile = (path, body) => {
+  fse.outputFile(
+    path,
+    body,
+    err => {
+      if (err) { showError(`Fail creating ${path}`) }
+      console.log(colors.gray(`Created ${path}`))
+    }
+  )
+}
+
+const createDir = (path) => {
+  fse.mkdirp(path, err => {
+    if (err) { showError(`Fail creating ${path}`) }
+    console.log(colors.gray(`Created ${path}`))
+  })
+}
+
+if (!PROJECT_NAME) { showError('the project name must be defined') }
+
+createDir(`${BASE_DIR}/${PROJECT_NAME}/components`)
+createDir(`${BASE_DIR}/${PROJECT_NAME}/demo`)
+
+writeFile(
+`${BASE_DIR}/${PROJECT_NAME}/components/README.md`,
+`# ${PROJECT_NAME} components
+<!-- Here put a description about your project -->`
+)
+
+writeFile(
+`${BASE_DIR}/${PROJECT_NAME}/package.json`,
+`{
+  "name": "${PROJECT_NAME}",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "phoenix": "rm -Rf node_modules && npm i",
```

Comment:
Our current studio was created with this one:

```
"phoenix": "rm -Rf node_modules && npm i && suistudio clean-modules && suistudio run-all npm i",
```

Maybe this one makes more sense? since it will be applied to all components

PD: similar for linting commands